code/game/objects/items/mountable_frames/paintings.dm

```diff
@@ -41,9 +41,8 @@ var/global/list/available_paintings = list(
 	item_state = "painting"
 	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/misc_tools.dmi', "right_hand" = 'icons/mob/in-hand/right/misc_tools.dmi')
 	flags = FPRINT
-	w_class = W_CLASS_MEDIUM
 	w_type = RECYK_WOOD
-	flammable = TRUE
+	flammable = FALSE
```

Comment:
wat

Reply:
Brain fart and thought this was the painting structure (which I don't want burning). Fixed